python/inputsForLimitExtraction/yieldFileAllYears.py

```diff
@@ -1,6 +1,24 @@
 from collections import OrderedDict
 
-beforeSel = False
+beforeSel   = False
+splitInputs = False
+
+if splitInputs and not beforeSel:
```

Comment:
Why do we need this?

Reply:
In case we split by years, with different directories and/or file names (that's what I did to reduce time).